.github/workflows/llm_unit_tests.yml

```diff
@@ -364,6 +364,7 @@ jobs:
           pip install -U langchain==0.0.184
           pip install -U chromadb==0.3.25
           pip install -U pandas==2.0.3
+          pip install -qU langchain langchain-community
```

Comment:
does it conflict with the version in Line364?

Reply:
Not for now. But in case this problem happens, I'll specify version numbers.